modules/core/src/main/java/org/apache/ignite/configuration/DataStorageConfiguration.java

```diff
@@ -957,6 +969,20 @@ public DataStorageConfiguration setWalCompactionEnabled(boolean walCompactionEna
         return this;
     }
 
+    /**
+     * @return ZIP level to WAL compaction.
+     */
+    public int getWalCompactionLevel() {
+        return walCompactionLevel;
+    }
+
+    /**
+     * @param walCompactionLevel New ZIP level to WAL compaction.
+     */
+    public void setWalCompactionLevel(int walCompactionLevel) {
+        this.walCompactionLevel = walCompactionLevel;
```

Comment:
I'm a bit late to the party, but I'm curious: why don't your return `this` like other setters?